tests/CMakeLists.txt

```diff
@@ -105,6 +106,7 @@ add_test(NAME keosd_auto_launch_test COMMAND tests/keosd_auto_launch_test.py WOR
 set_property(TEST keosd_auto_launch_test PROPERTY LABELS nonparallelizable_tests)
 add_test(NAME db_modes_test COMMAND tests/db_modes_test.sh WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 set_tests_properties(db_modes_test PROPERTIES COST 6000)
+set_property(TEST db_modes_test PROPERTY LABELS nonparallelizable_tests)
```

Comment:
Why is this needed? That test is designed to be parallelizable.

Reply:
It listens on some default TCP ports which cannot be shared with others.

Reply:
What ports? It's launched with `--http-server-address '' --p2p-listen-endpoint ''` which has historically disabled listening on all ports.

Reply:
revert the change in commit b068313ccfa8b3be9d9d70f861a7dd2eccc69b46